### DISCUSSION TITLE
How to show all failed scenarios from different feature files

### DISCUSSION THREAD

**Discussion by siva90144** [WasiqB/multiple-cucumber-html-reporter#554] [Ideas]
The report seems comprehensive, but it's worth considering a feature to easily view all the failed scenarios across different feature files.

For instance, if you have six feature files and each contains several scenarios, it would be beneficial to have a single view that aggregates all the failed scenarios, rather than needing to navigate through each feature file individually. This way, users could quickly identify and address all the failures at once, streamlining the debugging process. Such a feature could significantly enhance the usability and efficiency of your testing framework.

**Comment by WasiqB** [comment]
This feature sounds good, but I would keep this discussion open for upvotes from the community if they also feel if this feature would be useful.
